test/features/deploy_travel_agency.feature

```diff
@@ -0,0 +1,44 @@
+@quarkus
+@cr
+Feature: Deploy Travel agency service and verify its functionality
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed with Infinispan and Kafka operators
+    And "CR" install Kogito Data Index with 1 replicas
+    And "CR" deploy "travelapp-kogito-travel-agency.yaml" example service
```

Comment:
maybe add `file` instead of `service` ? or add it ?

Reply:
Well, the purpose is to deploy service from yaml which is located in examples folder.

So, maybe:
"CR" deploy service from example file "travelapp-kogito-travel-agency.yaml"

What do you think about it?

Reply:
I do like that one :)